setup.cfg

```diff
@@ -0,0 +1,38 @@
+[flake8]
+
+# Black compatibility.
+ignore = E203,E501,W503
+exclude = build,dist,env,.pytype
+
+
+[isort]
+
+# Settings for compatibility with black.
+line_length = 88
+use_parentheses = True
+multi_line_output = 3
+include_trailing_comma = True
+
+# Don't misclassify larq as a first-party import.
+known_third_party = larq
+
+skip =
+    build
+    dist
+    env
+    .pytype
+
+
+[pytype]
+
+inputs = .
+output = .pytype
+exclude =
+    build
+    dist
+    env
+    .pytype
+    examples
+# Keep going past errors to analyse as many files as possible.
+keep_going = True
+python_version = 3.7
```

Comment:
Since we also support Python 3.6, does it make sense to set it to 3.6?

Reply:
I just tried that and pytype but then get the error `CRITICAL Could not find a valid python3.6 interpreter in path (found Python 3.7.5)`. We could fix this on CI by installing 3.6 instead but then pytype would fail locally. So I've made everying 3.7 everywhere.

Reply:
Makes sense